### PR TITLE
feat: update storybook up to v7

### PR DESCRIPTION
**Pull request by NikitaCG**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        "targets": {
+          "chrome": 100
+        }
+      }
+    ],
+    "@babel/preset-typescript",
+    "@babel/preset-react"
+  ],
+  "plugins": []
+}
```
Comment (niktverd) [inline .babelrc:20]: Empty row at the end of the file

File: .npmrc [hunk 2]
```diff
@@ -1 +1,2 @@
-registry=https://registry.npmjs.org
\ No newline at end of file
+registry=https://registry.npmjs.org
+legacy-peer-deps=true
```
Comment (niktverd) [inline .npmrc:2]: Empty string at the end of the file

File: .nvmrc [hunk 3]
```diff
@@ -1 +1 @@
-14
\ No newline at end of file
+18
```
Comment (niktverd) [inline .nvmrc:1]: Empty string at the end of the file

Comment (NikitaCG) [reply .nvmrc]: fix all

**Review by niktverd — APPROVED** [review]

**Pull request merged**